.env.sample

```diff
@@ -29,3 +29,5 @@ NEXT_PUBLIC_IMAGE_DOMAINS="cms.dev.codeforafrica.org"
 
 # JWT secret key. This must match the GRAPHQL_JWT_AUTH_SECRET_KEY constant in wp-config.php.
 JWT_SECRET_KEY="your-secret-token"
+
+NEXT_PUBLIC_MULTISITE_PREFIX="pesayetu"
```

Comment:
Don't know who came up with this but:

1. Lets rename it to `WORDPRESS_MULTISITE_SUBDIRECTORY`. Only add NEXT_PUBLIC_ if we **_really_** need it on the front end, and
2. Check that it exists before using it [here](https://github.com/CodeForAfrica/PesaYetu/blob/88265ae5353cdc9a29e81a61917f324247b4defb/src/functions/menus/formatHeirarchialMenu.js#L21-L23) in case the deployment is not multisite or it uses domains

Reply:
also not related to this PR but will do so